java/README.md

```diff
@@ -0,0 +1,10 @@
+# PDAL-JNI Scala bindings
```

Comment:
Should call the Java bindings, since they are made to be accessed through Java or any JVM language. Would be good to make mention of the fact that we are coding in Scala, but these types are easily accessible in Java.